lib/key/utils.js

```diff
@@ -2,11 +2,14 @@ import { openpgp } from '../openpgp';
 import { serverTime } from '../serverTime';
 
 // returns promise for generated RSA public and encrypted private keys
-export const generateKey = (options) => {
+export async function generateKey(options) {
     options.date = typeof options.date === 'undefined' ? serverTime() : options.date;
     return openpgp.generateKey(options);
-};
-export const generateSessionKey = (algorithm) => openpgp.crypto.generateSessionKey(algorithm);
+}
+
+export async function generateSessionKey(algorithm) {
```

Comment:
`export const generateSessionKey = openpgp.crypto.generateSessionKey`
No need to wrap it again ;) and it's already async

Reply:
Thanks, I'll remove from the commit